src/store/model/ldlc.ts

```diff
@@ -0,0 +1,244 @@
+import {Store} from './store';
+
+export const Ldlc: Store = {
+  currency: '€',
+  labels: {
+    inStock: {
+      container: '.stock-1',
+      text: ['en', 'stock'],
```

Comment:
```suggestion
      container: '.stock',
      text: ['stock'],
```

Perhaps we just take this change?

Reply:
updated, I think it's ready to merge